### ISSUE TITLE
Bug in Mount nfs share with installation media

### ISSUE THREAD

**Issue by Rendanic** [oravirt/ansible-oracle#19]
This task has a problem, with the following configuration:
```
install_from_nfs: false
oracle_sw_copy: false
oracle_sw_unpack: false
```

This results in trying to mount the nfs, which is not needed in my configuration, because I still have the files unpacked in a local folder.

The fix is very simple:
old:
`    when: not oracle_sw_copy|bool and not oracle_sw_unpack|bool`

new:
`    when: install_from_nfs`
    
Both variables rely on `install_from_nf`s in `defaults/main.yml`:

```
  oracle_sw_copy: "{% if install_from_nfs==true %}false{% else %}true{% endif %}"
  oracle_sw_unpack: "{% if install_from_nfs==true %}false{% else %}true{% endif %}"
```

Should I create a pull request for you?

**Comment by oravirt** [comment]
Yes!

That was the usecase I didn't think about ;-)

**Comment by oravirt** [comment]
fixed via e76c874
